nitter.example.conf

```diff
@@ -33,6 +33,12 @@ tokenCount = 10
 # always at least `tokenCount` usable tokens. only increase this if you receive
 # major bursts all the time and don't have a rate limiting setup via e.g. nginx
 
+# Instead of guest_accounts.json, fetch guest accounts from an external URL.
+# Nitter will re-fetch accounts if it runs out of valid ones.
+guestAccountsUrl = ""    # https://example.com/download
+guestAccountsHost = ""   # defaults to nitter's hostname
+guestAccountsKey = ""    # a random string that will be used as a secret to authenticate against the URL
```

Comment:
Let's put this into a new section, and also remove the `tokenCount` + comment above since that's no longer needed. This section can be used for other variables in the future like whether to try fetching accounts by itself, if so which proxy/proxies to use, jsonl file location, and some other tweaks to the auth handling.

```suggestion
# Instead of guest_accounts.jsonl, fetch guest accounts from an external URL.
# Nitter will re-fetch accounts if it runs out of valid ones.
[GuestAccounts]
usePool = false    # enable fetching accounts from external pool
poolUrl = ""       # https://example.com/download
poolAuth = ""      # random secret string for authentication
poolId = ""        # defaults to nitter's hostname
poolAmount = 10000 # amount of accounts to request from pool
```

Reply:
done

Reply:
> poolAmount = 10000 # amount of accounts to request from pool

I did not implement this. twitterminator does not allow you to negotiate the count, not in public api...